docs/_index.md

```diff
@@ -0,0 +1,392 @@
+---
```

Comment:
Just 2c: users will start trying to edit this file possibly because there's on warning it is auto-generated. We can deal with it. Not sure if there's a content-neutral way to add this warning. Maybe something like meta_desc.

Reply:
This is already shipped in the bridge and will be pulled in next bridge update: https://github.com/pulumi/pulumi-terraform-bridge/pull/2493